dev_guide/builds/build_inputs.adoc

```diff
@@ -257,6 +257,14 @@ xref:../../install_config/http_proxies.adoc#configuring-hosts-for-proxies-using-
 a global proxy for Git cloning using Ansible].
 endif::[]
 
+[NOTE]
+====
+For Pipeline strategy builds, given the current restrictions with the Git plug-in for Jenkins, proxies
+defined in your `BuildConfig` cannot be honored.  You must configure the proxy in the Jenkins UI.
+Instruction on how to configure proxies via the Jenkins UI can be found
+link:https://wiki.jenkins-ci.org/display/JENKINS/JenkinsBehindProxy[`here.`]
```

Comment:
it's really the git proxy (from the buildconfig) that's not honored (for purposes of accessing your git repo containing your Dockerfile).

env var proxies defined may (or may not) still be respected by various steps of your pipeline, depending on what those steps do and the plugins that implement them.  I think we need to be more specific here.